.ci/Jenkinsfile

```diff
@@ -314,12 +315,26 @@ def generateFunctionalTestStep(Map args = [:]){
 
   def goArch = "amd64"
   def workerLabels = "${platform} && immutable"
+  def linuxIp = null
   if (platform == "arm64") {
     goArch = "arm64"
     // ARM is not ready as immutable workers
     workerLabels = "arm"
   }
 
+    if(platform.contains("windows")) {
+        workerLabels = "windows-2019"
+        // Boot up a linux VM to deploy Elastic Stack to be tested with a
+        // Windows/MacOS agent utilizing remote provider option
+        withNode(labels: "ubuntu-18.04") {
+            deleteDir()
+            unstash 'source'
+            sh script: "git clone https://github.com/elastic/e2e-testing", label: "Checking out e2e-testing for deployment script"
```

Comment:
I think the unstash step is already retrieving the source code of the repo. Therefore, this clone is not needed, and i could eventually start the runtime dependencies in an old state (i.e. testing aPR affecting the profile)

Reply:
I attempted without the unstash but the source code was never found, did I miss something here?